src/index.html

```diff
@@ -1,13 +1,71 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<!DOCTYPE html>
+</html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__content">
+
+    <main class="card" data-qa="card">
+      <section class="card__inner">
+        <section class="product">
+          <img
+          class="product__img"
+          src="images/imac.jpeg"
+          alt="product"
+          >
+        </section>
+
+        <h2 class="title card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <section class="card__product">Product code: 195434</section>
+
+        <section class="card__star review">
+          <ul class="stars review__stars review__stars--4">
```

Comment:
```suggestion
          <div class="stars review__stars review__stars--4">
```

якщо це `ul` то де елементи списку?